platform/app/services/create_connection_and_syncs_service.rb

```
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+
+  class CreateConnectionAndSyncsService
```

rubocop

[convention] Indentation of first line in file detected.

(Layout/InitialIndentation)